install.sh.in

```diff
@@ -80,28 +103,4 @@ podman run --rm --name sat-cpc \
 print_stage "Cleaning up install dependencies"
 clean-install-deps
 
-# Wait for the sat-config-import job to complete.
-config_import_job_timeout=300
```

Comment:
I just realized this will speed up our installation by up to 5 minutes and reduce time spent debugging this job if it fails to complete in time, so this is a nice added benefit!